node/pkg/aggregator/aggregator.go

```diff
@@ -51,13 +51,15 @@ func (n *Aggregator) Run(ctx context.Context) {
 	if err != nil {
 		log.Error().Str("Player", "Aggregator").Err(err).Msg("failed to get latest round id, setting roundId to 1")
 	} else if latestRoundId > 0 {
-		n.RoundID = latestRoundId
+		n.RoundID = latestRoundId + 1
```

Comment:
the PR description mentions implementing thread safe access to RoundID but seems like this is a somewhat major logic update? If not, pls, ignore this comment.

Reply:
it could look major but might not be necessary and I'm even thinking of removing mutex 🤔. even though round id is accessed and increased through leader job which is in go thread, it is theoretically unlikely to happen since this is the only place where it's been accessed and increased, and it's being called every 400ms through ticker. concurrent access is not likely to happen

Reply:
pr name doesn't include this start round id update lol , will update